integration/tests/sample_data_test.go

```diff
@@ -0,0 +1,74 @@
+// +build integration
+
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+)
+
```

Comment:
Question: What are your thoughts on additional test data? Are you expecting this implementation to grow?

Reply:
We also will definitely need test data for resource types other than EC2 Instance.

Reply:
ok, just wanted to make sure i follow pattern